docs/streams/developer-guide/testing.html

```diff
@@ -73,67 +71,55 @@ <h2><a class="toc-backref" href="#testing-topologytestdriver" title="Permalink t
                 You can use the test driver to verify that your specified processor topology computes the correct result
                 with the manually piped in data records.
                 The test driver captures the results records and allows to query its embedded state stores.
-            <pre>
-// Processor API
+            <pre class="line-numbers"><code class="language-text">// Processor API
 Topology topology = new Topology();
 topology.addSource("sourceProcessor", "input-topic");
 topology.addProcessor("processor", ..., "sourceProcessor");
-topology.addSink("sinkProcessor", "result-topic", "processor");
+topology.addSink("sinkProcessor", "output-topic", "processor");
 // or
 // using DSL
 StreamsBuilder builder = new StreamsBuilder();
-builder.stream("input-topic").filter(...).to("result-topic");
+builder.stream("input-topic").filter(...).to("output-topic");
```

Comment:
Hmm, is this supposed to match the output topic in L103 below? That one says "result-topic".

Reply:
good catch! Thanks